Yank/pipeline.py

```diff
@@ -1852,7 +1852,7 @@ def extend_list_of_waters(leap_parameters):
                                                       _OPENMM_LEAP_SOLVENT_FILES_MAP[solvent_model]))
                 logger.warning(solvent_warning)
             leap_solvent_model = _OPENMM_LEAP_SOLVENT_MODELS_MAP[solvent_model]
-            clearance = float(solvent['clearance'].value_in_unit(unit.angstroms))
+            clearance = solvent['clearance'].value_in_unit(unit.angstroms)
```

Comment:
The new `openmmtools` is out, but there are still a few errors. Some of them I've fixed them in #1168, but I think you'll have to change this line to
```python
clearance = solvent['clearance']
```
(or just pass it directly to the function below) because `value_in_unit` still returns a unit-less numpy float.

Reply:
tleap.solvate() already checks that clearance will be written in the script in angstroms:

```
1603         self.add_commands('{} {} {} {} iso'.format(solvate_command, unit_name,
1604                                                    solvent_model, str(clearance.value_in_unit(unit.angstroms))))
```

As you suggest `clearance = solvent['clearance']` is enough.
I am going make the change.
Thanks!